testing/ncids-css-testing/stories/uswds-native/.editorconfig

```diff
@@ -0,0 +1,10 @@
+# Reset editorconfig for uswds files
+root = true
+
+[*.{json,js}]
```

Comment:
This should be json and twig, not js. We own the js and it should follow our standards.